### PR TITLE
simplify servlet loggers a bit

### PR DESCRIPTION
**Pull request by rvanheest**
removed calling certain methods twice in a row

@DANS-KNAW/easy for review

### CODE REVIEW

File: src/main/scala/nl.knaw.dans.easy.deposit/logging/RequestLogFormatter.scala [hunk 1]
```diff
@@ -51,7 +52,7 @@ trait RequestLogFormatter extends CookieFormatter {
     }
   }
 
-  private def getHeaderMap: HeaderMap = {
+  private def getHeaderMap(request: HttpServletRequest): HeaderMap = {
```
Comment (jo-pol) [inline src/main/scala/nl.knaw.dans.easy.deposit/logging/RequestLogFormatter.scala]: Why not use the implicit provided by ScalatraBase?

Comment (jo-pol) [reply src/main/scala/nl.knaw.dans.easy.deposit/logging/RequestLogFormatter.scala]: I would prefer you fixed your development environment.

Comment (jo-pol) [reply src/main/scala/nl.knaw.dans.easy.deposit/logging/RequestLogFormatter.scala]: Tried to synchronize and `mvn clean install` all projects?

**Comment by rvanheest** [comment]
closing since this is replaces by #100

**Pull request closed without merging**